src/web/src/components/Posts/Post.tsx

```diff
@@ -193,6 +203,10 @@ const PostComponent = ({ postUrl }: Props) => {
   // Grab the post data from our backend so we can render it
   const { data: post, error } = useSWR<Post>(postUrl);
   const [expandHeader, setExpandHeader] = useState(false);
+  // If post exists, extract the blog platform name
```

Comment:
Move this down to ~248 where you know post exists for sure.  Don't bother if it doesn't.  Also, I would suggest you should do it all in a function that you call vs. using a variable:

This needs a space between.  I'd do it like this:

```jsx
className={`telescope-post-content ${extractBlogClassName(post.url)}`}
```